### DISCUSSION TITLE
Previously working, now getting this error

### DISCUSSION THREAD

**Discussion by shortforrichard** [DIGITALCRIMINALS/OnlyFans#539] [General]
Scraper was previously working, but yesterday I started getting this error. Any ideas how to troubleshoot it?

![image](https://user-images.githubusercontent.com/110058313/181086147-cfc1d3b8-9e44-4054-841c-695b2eba141b.png)

**Comment by shortforrichard** [comment]
this solved it
https://github.com/DIGITALCRIMINALS/OnlyFans/issues/518#issuecomment-1193026208

**Reply by mwill39014** [reply]
I started getting a similar error:
![OF Scraper Error 2022 07 30](https://user-images.githubusercontent.com/57028629/182242023-0bc87212-5be1-48fd-9cf4-86ac7d802fff.jpg)

Came here, found this thread, followed the above link, made the edit to line 270, now I get this error:
![OF Scraper Error 2022 07 30 2](https://user-images.githubusercontent.com/57028629/182242293-aad8f003-30e9-4dc5-8b1a-266ab3fc3405.jpg)
